tools/project_manager/edit_common.inc

```diff
@@ -78,7 +78,8 @@ function language_list($language)
 function genre_list($genre)
 {
     $genres = load_genre_translation_array();
-    echo "<select name='genre'>";
+    echo "<select name='genre'>\n";
+    echo "<option value=''>&nbsp;</option>\n";
```

Comment:
For the fields that require a value, we might consider doing something like this instead:
```
"<option value='' disabled>" . _("Select one") . "</option>"\n
```

That way users know what fields are required by looking at them instead of clicking save and changing something until it works.

Reply:
Good idea, I'll do that.

Reply:
That's a good idea.

Would it be reasonable to put shadow text in the "Name of Work" and "Author's Name" input boxes that says "required field". Not that anyone is likely to try to leave at least the title field out. The four fields that currently will generate an error are Name of Work, Author, Project manager, and Primary Language (and once this gets fixed, Genre).

Project manager is auto-filled with the username of the PM who created the project, so the only time that error is triggered is when a squirrel removes the username without adding another.

Reply:
Giving the <select> and <input> a required attribute has the required effect.